transport/grpc/handler_test.go

```diff
@@ -0,0 +1,73 @@
+package grpc
+
+import (
+	"testing"
+
+	"fmt"
+	"net/url"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetServiceAndProcedureFromMethod_Base(t *testing.T) {
+	method := "foo/bar"
+
+	service, procedure, err := getServiceAndProcedureFromMethod(method)
+
+	assert.Equal(t, nil, err)
+	assert.Equal(t, "foo", service)
+	assert.Equal(t, "bar", procedure)
+}
+
+func TestGetServiceAndProcedureFromMethod_ExtraSlash(t *testing.T) {
+	method := "/foo/bar"
+
+	service, procedure, err := getServiceAndProcedureFromMethod(method)
```

Comment:
Instead of duplicating the test code, use test tables.

Do something like,

```
tests := []struct{
    method string
    
    wantService string
    wantProcedure string
    wantError error
}{
    {
        method: "/foo/bar",
        wantService: "foo",
        wantProcedure: "bar",
    },
}

for _, tt := range tests {
    // ...
}
```

https://github.com/golang/go/wiki/TableDrivenTests